src/main/java/core/basesyntax/library/Injector.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.library;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.factory.Factory;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+public class Injector {
+    public static Object getInstance(Class clazs) throws NoSuchMethodException,
+            IllegalAccessException, InvocationTargetException, InstantiationException {
+
+        Constructor constructor = clazs.getDeclaredConstructor();
+        Object instance = constructor.newInstance();
+
+        Field[] declaredFields = clazs.getDeclaredFields();
+        for (Field field : declaredFields) {
+            if (field.getAnnotation(Inject.class) != null) {
```

Comment:
maybe better use for check isAnnotationPresent? but i don't sure.

Reply:
Ok. Done